CHANGELOG.md

```diff
@@ -660,6 +667,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [v0.0.76]: https://github.com/agentuity/cli/compare/v0.0.75...v0.0.76
 [v0.0.75]: https://github.com/agentuity/cli/compare/v0.0.74...v0.0.75
 [v0.0.74]: https://github.com/agentuity/cli/compare/v0.0.73...v0.0.74
+[v0.0.134]: https://github.com/agentuity/cli/compare/v0.0.133...v0.0.134
```

Comment:
for the love of zeus, these are semantic versions and they should be ordered by the most recent release before older releases. please move this link to be the first link in the list ⚡️